### PR TITLE
Add documentation for FML imports and includes

### PR DESCRIPTION
**Pull request by jhugman**
## Description (optional)

Closes [EXP-2340](https://mozilla-hub.atlassian.net/browse/EXP-2340).

### CODE REVIEW

File: docs/deep-dives/specifications/fml-spec.mdx [hunk 1]
```diff
@@ -457,11 +477,11 @@ In this way, app developers can use the feature manifest as a replacement for ot
 
 Primitive types supported:
 
- * `String` or `string`
- * `Boolean`, `Bool` or `boolean`. Both Kotlin and Swift's spelling is supported.
```
Comment (jaredlockhart) [inline docs/deep-dives/specifications/fml-spec.mdx:461]: Will deprecating these aliases cause breakage to existing manifests?

**Review by jaredlockhart — APPROVED** [review]
This is amazing @jhugman .  Very readable and comprehensive.  Thank you!

File: docs/deep-dives/specifications/fml-spec.mdx [hunk 2]
```diff
@@ -481,17 +501,54 @@ features:
...
+<TabItem value="kotlin">
+
+Values for `Text` and `Image` are strings which must correspond to the following format:
+
+`[a-z][a-z_0-9]*`
+
+The lookup uses the `R` class.
+
+</TabItem>
+</Tabs>
```
Comment (jeddai) [inline docs/deep-dives/specifications/fml-spec.mdx:538]: It's kinda difficult to tell where this tab block ends when rendered, could it be worth it to add a horizontal rule or something else to more clearly separate it?

https://user-images.githubusercontent.com/4909307/199591921-b2b40d81-1727-4ad7-9de1-f591a4cc3205.mov

File: docs/deep-dives/specifications/fml-spec.mdx [hunk 3]
```diff
@@ -481,17 +501,54 @@ features:
...
+For Android only:
+
+Android uses integer identifiers stored in the `R` class. Sometimes these identifiers are what is required.
+
+In the above example:
+
+```kotlin
+val heroImage = FxNimbus.features.upgradeMessage.value().heroImage
+val resId: Int = heroImage.resourceId
+val image: Drawable = heroImage.resource
```
Comment (jeddai) [inline docs/deep-dives/specifications/fml-spec.mdx:551]: I think this section maybe should be in the kotlin section of the above tab block? The part about the key not resolving is in the swift block already, and the "For Android only" section wouldn't apply to Swift code.

File: docs/deep-dives/specifications/fml-paths.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  "my/onboarding": "./Onboarding/nimbus-manifests"
+}
+```
+
+### Multiple repo files
+
+The `nimbus-fml` command line can accept multiple repo-files at a time, resolving `@` links as it goes.
+
+```sh
+nimbus-fml generate --repo-path ./app-structure.json --repo-path @mozilla-mobile/dependency-versions/v102.0.json --language swift ./input.fml.yaml output-dir
```
Comment (jeddai) [inline docs/deep-dives/specifications/fml-paths.md:63]: I think the `repo-path` arguments are meant to be `repo-file`.
```suggestion
nimbus-fml generate --repo-file ./app-structure.json --repo-file @mozilla-mobile/dependency-versions/v102.0.json --language swift ./input.fml.yaml output-dir
```

**Review by jeddai — CHANGES_REQUESTED** [review]
I've got a few thoughts, but this is looking great!

File: docs/deep-dives/specifications/fml-paths.md (hunk 4 above)
Comment (jeddai) [reply docs/deep-dives/specifications/fml-paths.md:63]: This is the only one of my comments I would definitely prefer eyes on before this is merged

File: docs/deep-dives/specifications/fml-spec.mdx (hunk 1 above)
Comment (jhugman) [reply docs/deep-dives/specifications/fml-spec.mdx:461]: No, they were never supported :(

File: docs/deep-dives/specifications/fml-spec.mdx (hunk 2 above)
Comment (jhugman) [reply docs/deep-dives/specifications/fml-spec.mdx:538]: Yes!

**Pull request merged**